serializations.md

```diff
@@ -121,6 +121,7 @@ subsequent sections divided by object.
 | Channel settle transaction | 56 |
 | Channel off chain transaction | 57 |
 | Channel | 58 |
+| POI | 60 |
```

Comment:
I see documented because of https://github.com/aeternity/epoch/pull/1202#discussion_r191644134
Ok.